app/src/main/java/org/fossasia/openevent/app/module/ticket/create/CreateTicketPresenter.java

```diff
@@ -26,10 +26,17 @@
     public CreateTicketPresenter(ITicketRepository ticketRepository) {
         this.ticketRepository = ticketRepository;
         LocalDateTime current = LocalDateTime.now();
+        String startDate = DateUtils.formatDateToIso(current);
+        ticket.getSalesStartsAt().set(startDate);
 
-        String isoDate = DateUtils.formatDateToIso(current);
-        ticket.getSalesStartsAt().set(isoDate);
-        ticket.getSalesEndsAt().set(isoDate);
+        LocalDateTime salesEndTime = current.plusDays(10);
+        LocalDateTime eventEndTime = DateUtils.getIsoOffsetTimeFromTimestamp(ContextManager.getSelectedEvent().getEndsAt().get());
```

Comment:
@iamareebjamal Can you please help here ? 
I am trying to make sure that the sales end time doesn't go beyond the event's end time. To do that i have to use 
```ContextManager```'s static method ```getSelectedEvent``` to get the event's end time. 
Now Mockito doesn't support static methods. That's why the build fails. How to go about doing this ? Should i use PowerMockito for testing ?

Reply:
Use ContextManager's `setSelectedEvent`